kse/src/org/kse/gui/actions/resources.properties

```diff
@@ -272,6 +272,14 @@ GenerateSecretKeyAction.statusbar                             = Generate a Secre
 GenerateSecretKeyAction.text                                  = Generate Secret Key
 GenerateSecretKeyAction.tooltip                               = Generate Secret Key
 
+GenerateDHParametersAction.GenerateDHParameters.Title               = Generate DH Parameters
+GenerateDHParametersAction.DHParametersGenerationSuccessful.message = DH Parameters Generation Successful.
+GenerateDHParametersAction.accelerator                           = D
+GenerateDHParametersAction.statusbar                             = Generate DH Parameters in the active KeyStore
```

Comment:
"in the active KeyStore" should be removed

Reply:
Updated.